packages/react-components/src/accordion/src/Accordion.css

```diff
@@ -117,5 +119,5 @@
 
 .o-ui-accordion-trigger:focus::before,
 .o-ui-accordion-trigger-focus::before {
-    border-color: var(--primary-500);
+    border-color: var(--brand-border);
```

Comment:
why "brand" instead of "primary" ?

Reply:
Primary is used in a hierarchy/importance context.

Reply:
As discussed in a call yesterday, this might be confusing.